@here/harp-mapview/lib/MapView.ts

```diff
@@ -1463,6 +1477,36 @@ export class MapView extends THREE.EventDispatcher {
         this.update();
     }
 
+    /**
+     * Get currently presented political point of view - the country code.
+     *
+     * @note Country code is stored in ISO 3166-1 alpha-2 standard.
+     * @return Country code or undefined if default
+     * (majorly accepted) point of view is used.
+     */
+    get politicalView(): string | undefined {
+        return this.m_politicalView;
+    }
+
+    /**
+     * Set the political view (country code) to be used when rendering disputed features (borders).
+     *
+     * @note Country code should be encoded in ISO 3166-1 alpha-2 standard.
+     * @param pov The code of the country which point of view should be presented,
+     * if `undefined` "defacto" or most widely accepted point of view will be presented.
+     */
+    set politicalView(pov: string | undefined) {
+        if (this.m_politicalView === pov) {
+            return;
+        }
+        this.m_politicalView = pov;
+        this.m_tileDataSources.forEach((dataSource: DataSource) => {
+            dataSource.setPoliticalView(pov);
+        });
+        this.clearTileCache();
```

Comment:
Already commented.
Not needed here.
It's DataSource responsibility to update it's Tiles in implementation of DataSource.setPoliticalView. For some DS no invalidation is needed.

Reply:
Ok removed.